### ISSUE TITLE
move and close methods

### ISSUE THREAD

**Issue by illmillrig** [illmillrig/BlurBetween#5]
borderless UI needs a way to close,  also would like to override moveEvent() to allow moving the UI position limited to the bounds of the viewport by cntrl click dragging.  Will need to update moveToPosition() to keep the UI's position weighted between the viewports 4 corners. so that when stuck to the right side of the viewport, it remains there during viewport resizing.

**Comment by artintime** [comment]
Yes it needs a way to close becasue at the moment it is stuck right in the top of the viewport and can't close it at all. Any ideas?  Great plugin by the way!
Thanks

**Comment by illmillrig** [comment]
you can just unload the plugin in the meantime
